src/BuildScriptGenerator/Node/NodeBashBuildSnippet.sh.tpl

```diff
@@ -115,27 +116,28 @@ npm pack
 
 if [ "$copyOnlyProdModulesToOutput" == "true" ]
 then
-	# Rename folder having all node_modules as we want only prod dependencies
-	# to be synced with destination directory
-	mv node_modules $allModulesDirName
+	echo
+	echo "Copy '$SOURCE_DIR/node_modules' with all dependencies to '$SOURCE_DIR/$allModulesDirName'..."
+	rsync -rtE --links "node_modules/" "$allModulesDirName" --delete
```

Comment:
are we deleting links/files in destination?

Reply:
We are only deleting files/links in the target directory that are not present in source directory